erpnext/e_commerce/doctype/e_commerce_settings/e_commerce_settings.py

```diff
@@ -93,3 +164,9 @@ def check_shopping_cart_enabled():
 
 def show_attachments():
 	return get_shopping_cart_settings().show_attachments
+
+def home_page_is_products(doc, method):
```

Comment:
```suggestion
def set_home_page(doc, method):
```
Also, can we add an alert to inform the user about this internal magic to avoid homepage reset frustration?

Reply:
@surajshetty3416 I'll remove this checkbox and this reset feature. Users can simply set it in Website Settings